pylint/checkers/exceptions.py

```diff
@@ -59,7 +60,7 @@ def _is_raising(body: list[nodes.NodeNG]) -> bool:
     return any(isinstance(node, nodes.Raise) for node in body)
 
 
-OVERGENERAL_EXCEPTIONS = ("BaseException", "Exception")
+OVERGENERAL_EXCEPTIONS = ("builtins.BaseException", "builtins.Exception")
```

Comment:
Could you move this to L248? We don't instantiate this class more than once so this should just be a value in the `dict` like all other options do.